distribution/std-lib/Table/src/Data/Column.enso

```diff
@@ -13,13 +13,16 @@ type Column
     display : Integer -> Text
     display show_rows=10 =
         java_col = this.java_column
+        index = this.java_column.getIndex []
```

Comment:
I don't understand the question, can you rephrase?

Reply:
Sorry, my sentence generation engine must have crahsed.

What I meant is, why don't you use `java_col` in this line like in all the other lines but instead refer to the full `this.java_column` which is equivalent to `java_col` here. Just asking because it was inconsistent.